cg/apps/sequencing_metrics_parser/models/bcl_convert.py

```diff
@@ -0,0 +1,62 @@
+from pydantic import BaseModel, BaseConfig, Field
+import xml.etree.ElementTree as ET
+
+
+class BclConvertQualityMetrics(BaseModel):
+    """Model for the BCL Convert quality metrics"""
+
+    lane: int
```

Comment:
I think you can send in the dict to the model like this BclConvertQualityMetrics(**YOUR_DICT) See pydantic docs.

Then the keys in the dict will be used as args. So exploit the alias function in pydantic: `Field(..., alias="DICT_IN_KEY")` to rename to logical python attributes.

Use constant class to define the aliases going in to not hard code the keys in the model.

Similar to
```
 flowcell_id: str = Field(..., alias=SampleSheetHeaderColumnNames.FLOW_CELL_ID.value)
```